download_models/downloadOVMSModels.sh

```diff
@@ -196,17 +198,6 @@ else
     getProcessFile $yolov5s $pipelineZooModel"yolov5s-416" $yolojson $yolov5s $modelPrecisionFP16INT8
 fi
 
-yolov5ModelFile="${PWD}/$yolov5s/$modelPrecisionFP32INT8/1/$yolov5s.bin"
-echo "$yolov5ModelFile"
-if [ -f "$yolov5ModelFile" ]; then
-    echo "yolov5s $modelPrecisionFP32INT8 model already exists, skip downloading..."
-else
-    echo "Downloading yolov5s $modelPrecisionFP32INT8 models..."
-    # Yolov5s FP32_INT8
-    getModelFiles $yolov5s $pipelineZooModel"yolov5s-416_INT8" $modelPrecisionFP32INT8
-    getProcessFile $yolov5s $pipelineZooModel"yolov5s-416" $yolojson $yolov5s $modelPrecisionFP32INT8
-fi
-
```

Comment:
why are we deleting yolov5 FP32INT8 downloading:   do we not need it now?

Reply:
> why are we deleting yolov5 FP32INT8 downloading: do we not need it now?

that's duplicated one, not sure why we had duplicated download yolov5 model in the same file.

Reply:
not the duplicated one, has different precision from the one above